### PR TITLE
Markdown file vertical format update

### LINKED ISSUE #911: Markdown file vertical format update
### Overview
We need to update the Vertical values format to include single quotes  so that we maintain a consistent format across the various verticals

### Action Items per Markdown file 
- [x] Update where it currently says vertical:  value to vertical:   'value'  

For Example, if it says the below 
```
vertical: Diversity, Equity & Inclusion 
```
Change it to
```
vertical: 'Diversity, Equity & Inclusion' 
```

### Project Markdown files to update
- [x] 311
- [x] Engage
- [x] Food Oasis
- [x] Lucky Parking
- [x] Civic Tech Index
- [x] 100 Automations
- [x] Civic Opportunity Project
- [x]  Civic Tech Structure
- [x] VRMS

### Resources/Instructions
[Citizen Engagement Vertical Information](https://github.com/hackforla/website/issues/891)
[Civic Tech Infrastructure Vertical Information](https://github.com/hackforla/website/issues/892)

- This is where we add any & all available resources. Do not shy away from adding things. If you think your audience already has it or might not need it, don't let that stop you. Add it anyway! It is very time effective to have all resources available in a single place.

### PR DESCRIPTION
**Pull request by trtincher**
Fixes #911

Vertical values reformatted to be wrapped in single quotes in the following .md files:
 311
 Engage
 Food Oasis
 Lucky Parking
 Civic Tech Index
 100 Automations
 Civic Opportunity Project
 Civic Tech Structure
 VRMS

For Example, if it says the below

`vertical: Diversity, Equity & Inclusion `

Change it to

`vertical: 'Diversity, Equity & Inclusion' `

### CODE REVIEW

**Comment by trtincher** [comment]
Made a couple of corrections after making the pull request in food-oasis.md and civic-tech-structure.md to correct for auto-formatting issues. I have turned off my auto formatting to prevent similar problems in the future.

File: _projects/food-oasis.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ leadership:
   - name: Tina Moh
     role: UX/Service Lead
     links:
-      slack: https://hackforla.slack.com/team/UP4AZM459'
+      slack: "https://hackforla.slack.com/team/UP4AZM459"
```
Comment (ItsLaszlo) [inline _projects/food-oasis.md:19]: Should this be in single quotes?

**Review by ItsLaszlo — COMMENTED** [review]
One edit possibly needed on line 19 in _projects/food-oasis.md.

File: _projects/food-oasis.md (hunk 1 above)
Comment (trtincher) [reply _projects/food-oasis.md:19]: So actually no, not if it is to match the rest of the page. However, this page is using different formatting from most of the others. So, I might bring this up on Tuesday as that seems a bit odd.

Comment (trtincher) [reply _projects/food-oasis.md:19]: @martham0 Yeah civic-tech-structure.md is also using quote formatting different from  the rest which I am now realizing is what caused me some trouble in the first place. I imagine what I should do is make these two pages match the rest but I will run it by the team on Tuesday to be sure that is the right move.

**Review by daniellex0 — APPROVED** [review]
Everything looks good! And re: your discussion above @martham0 and @trtincher , yes there are definitely a bunch of quote formatting inconsistencies in these md files that I've noticed too- agreed it's something good to discuss on Tuesday

**Review by ItsLaszlo — APPROVED** [review]
Everything looks good.

**Pull request merged**